app/models/response.rb

```diff
@@ -102,9 +113,10 @@ def initialize_invitation_token!
     # created_at is always when the object was last used. Also, if we don't first destroy the
     # invitation_token, then it will set a different token than what we're giving (since tokens have
     # to be unique).
-    token = invitation_token&.token
+    pre_token = invitation_token&.token_plain
     invitation_token&.destroy
-    create_invitation_token!(token: token)
+    create_invitation_token! token: pre_token
```

Comment:
`pre_token` kan toch ook nil zijn hier? Gaat dit dan goed of initialiseer je dan een leeg token?

Reply:
nee dan pakt de after_initialize van de token m op:

```ruby
a = Response.create(protocol_subscription_id: 1, measurement_id: Measurement.first.id, open_from: Time.zone.now)
a.invitation_token = nil
a.save!
a.initialize_invitation_token!
=> "kjb8"
```